bin/node/executor/tests/pov_limit_weights.rs

```diff
@@ -0,0 +1,42 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use frame_support::{pallet_prelude::Weight, traits::Hooks};
+use kitchensink_runtime::{PovLimit, Runtime, System};
+use pallet_pov_limit::WeightInfo;
+
+pub mod common;
+use self::common::*;
+
+#[test]
+fn expected_weight_same_as_actual() {
+	let mut t = new_test_ext(compact_code_unwrap());
+
+	let actual_weight = <Runtime as pallet_pov_limit::Config>::WeightInfo::on_idle();
```

Comment:
For some reason, the `ref_time` here is `~250_000_000` while in the `weight.rs` of the pallet it is `206_000_000` and that is what it is supposed to be. Not sure why am I getting a greater weight here.

Reply:
You can add this test directly to the pallet tests. And maybe use `Perbill` for nicer percent renderings.